src/frontend/src/layouts/PageTitle/NavUserMenu.tsx

```diff
@@ -18,11 +18,20 @@ import ListItemIcon from '@mui/material/ListItemIcon';
 import SettingsIcon from '@mui/icons-material/Settings';
 import LogoutIcon from '@mui/icons-material/Logout';
 import { canAccessAdminTools } from '../../utils/users';
+import { Stack, useTheme } from '@mui/system';
+import { Typography } from '@mui/material';
+import { fullNamePipe } from '../../utils/pipes';
+import { useCurrentUser } from '../../hooks/users.hooks';
 
-const NavUserMenu: React.FC = () => {
+interface NavUserMenuProps {
+  open?: boolean;
+}
+
+const NavUserMenu: React.FC<NavUserMenuProps> = ({ open }) => {
```

Comment:
this popup should open to the right and not on top of the button when you click on it. Also, see how this looks in the top left, I think it might actually fit